Source/BugsnagKSCrashSysInfoParser.h

```diff
@@ -10,5 +10,4 @@
 
 #define PLATFORM_WORD_SIZE sizeof(void*)*8
```

Comment:
To my mind this should now live in the `BugsnagDevice.h` header.

Reply:
This would make the macro publicly accessible which wouldn't be desirable as it might need changing/removing in future. Additionally this macro is also used in `BSGOutOfMemoryWatchdog`, which doesn't currently have knowledge about `BugsnagDevice`

Reply:
Moved to `BSGOutOfMemoryWatchdog.h` - as this is non-public and the only other place that uses it